### ISSUE TITLE
SVG needs Title and Description Attributes

### ISSUE THREAD

**Issue by mve83** [pojome/one-click-accessibility#72]
Hello

My pages are scored lower than necessary becasue the SVG icons are missing a title and description attribute. Its worth a couple of percent in the overall page score on webaccessibility tests. I've manually added them now and my page went from 95% to 97%.

Please add these as it seems an accessibility plugin is doing inaccessible things
